gpytorch/lazy/mul_lazy_variable.py

```diff
@@ -102,13 +114,25 @@ def _matmul(self, rhs):
             m = rhs.size(-1)
             # Now implement the formula (A . B) v = diag(A D_v B)
             left_res = (rhs.unsqueeze(-2) * left_root.unsqueeze(-1))
-            left_res = left_res.view(n, rank * m) if batch_size is None else left_res.view(batch_size, n, rank * m)
+            left_res = left_res.view(
+                n, rank * m
+            ) if batch_size is None else left_res.view(
+                batch_size, n, rank * m
+            )
             left_res = self.right_lazy_var._matmul(left_res)
-            left_res = left_res.view(n, rank, m) if batch_size is None else left_res.view(batch_size, n, rank, m)
+            left_res = left_res.view(
```

Comment:
There are a few things like this that can be written more nicely like

```
left_res = (
    left_res.view(n, rank, m)
    if batch_size is None
    else left_res.view(batch_size, n, rank, m)
)

```
but we can fix those later.